README.md

```diff
@@ -282,6 +282,8 @@ Read [CONTRIBUTING.md](/CONTRIBUTING.md) to learn how to add your own repos.
 
 *Solutions to AoC in PHP.*
 
+* [XonneX/adventofcode2020]
```

Comment:
```suggestion
* [XonneX/adventofcode2020](https://github.com/XonneX/adventofcode2020)
```

Reply:
Shouldn't we follow the instructions? https://github.com/Bogdanp/awesome-advent-of-code/blob/master/CONTRIBUTING.md
Don't want to break the GitHub Action.
If the instructions are outdated I can take this in the PR as well.

Reply:
The instructions are correct and so is your PR.